Using `google-services.json-example` crashes the app

**Describe the bug**
Using `google-services.json-example` file as a `google-services.json` will crash the app with output:

```
 Caused by: java.lang.IllegalArgumentException: Please set your project ID. A valid Firebase project ID is required to communicate with Firebase server APIs: It identifies your project with Google.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the project
2. Run `cp gradle.properties-example gradle.properties`
3. Install app, e.g. `./gradlew installJalapenoDebug`
4. Run the app
5. Observe crash

**Expected behavior**
The app should not crash.

--- a/WooCommerce/google-services.json-example
+++ b/WooCommerce/google-services.json-example
@@ -1,7 +1,7 @@
 {
   "project_info": {
     "project_number": "",
-    "project_id": ""
+    "project_id": "project_id"
   },
   "client": [
     {
@@ -27,7 +27,7 @@
       ],
       "api_key": [
         {
-          "current_key": ""
+          "current_key": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
         }
       ],
       "services": {
@@ -69,7 +69,7 @@
       ],
       "api_key": [
         {
-          "current_key": ""
+          "current_key": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
         }
       ],
       "services": {
@@ -111,7 +111,7 @@
       ],
       "api_key": [
         {
-          "current_key": ""
+          "current_key": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
         }
       ],
       "services": {

